testfile2.py

```
@@ -0,0 +1,818 @@
+    Updates context dictionary with course info.
+    """
+    context['full_course_image_url'] = request.build_absolute_uri(course_image_url(course))
+    course_title_from_cert = context['certificate_data'].get('course_title', '')
+    accomplishment_copy_course_name = course_title_from_cert if course_title_from_cert else course.display_name
+    context['accomplishment_copy_course_name'] = accomplishment_copy_course_name
+    course_number = course.display_coursenumber if course.display_coursenumber else course.number
+    context['course_number'] = course_number
+    try: # Adding support if course_start and course_end exists
+        context['course_start'] = course.start.strftime("%b %Y")
+        context['course_end'] = course.end.strftime("%b %Y")
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)